.github/workflows/release.yml

```diff
@@ -33,14 +35,11 @@ jobs:
         run: | # If the neofs module is to be published in the future, remove the -x :neofs:publishToSonatype
           ./gradlew --info -x test -x :gradle-plugin:publishToSonatype -x :neofs:publishToSonatype publishToSonatype closeAndReleaseSonatypeStagingRepository
           ./gradlew :gradle-plugin:publishPlugin -Pgradle.publish.key=$GRADLE_PUBLISH_KEY -Pgradle.publish.secret=$GRADLE_PUBLISH_SECRET
-      - name: Release Snapshot
-        if: github.event.inputs.snapshot == 'true'
+      - name: Release Snapshot Version
+        if: inputs.release-type == 'Snapshot'
         env:
           ORG_GRADLE_PROJECT_signingKey: ${{ secrets.GPG_KEY_ARMORED }}
           ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.GPG_KEY_PASSWORD }}
           ORG_GRADLE_PROJECT_sonatypeUsername: ${{ secrets.SONATYPE_USERNAME }}
           ORG_GRADLE_PROJECT_sonatypePassword: ${{ secrets.SONATYPE_PASSWORD }}
-        run: | # If you want to release a snapshot of gradle-plugin, remove the -x :gradle-plugin:publishToSonatype
-          ./gradlew --info -x test publishToSonatype -Psnapshot 
-
-
+        run: ./gradlew --info -x test publishToSonatype -Psnapshot
```

Comment:
I probably made a mistake copying that comment or the command. The `-x :gradle-plugin:publishToSonatype` flag prevents the plugin from being published as a maven artifact. I don't think that it is of any use if it is published as a Maven artifact, only if it is published on Gradle Plugin Repo. The command in the "Official Release" workflow does that 
```
./gradlew :gradle-plugin:publishPlugin -Pgradle.publish.key=$GRADLE_PUBLISH_KEY -Pgradle.publish.secret=$GRADLE_PUBLISH_SECRET
```
Thus, I think my intention was to not publish the gradle-plugin as a Maven artifact even for snapshots.

Oooh, but then I realized that we actually do use the Gradle Plugin Snapshot as a Maven artifact, as described in `DEVELOPMENT.md`. Thus, I removed the `-x :gradle-plugin:publishToSonatype` from the command here and forgot to adapt the comment. Mystery solved. Your change is correct.